resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -60,6 +61,23 @@ object DbReader {
           )"""
       .query[Runtime]
 
+  val deletingRuntimeQuery =
```

Comment:
Seems like a different name is warranted since this gets runtimes in `Stopped` and `Running` as well. Maybe name it or leave a comment to indicate the usage as opposed to the statuses it includes.

Reply:
I am not sure I understand, the goal is to grab runtimes marked as `deleting` in leo db, but would show as Running or Stopped in Google. Similar to the deletedRuntimeQuery and the erroredRuntimeQuery

Reply:
AoU re-uses runtime names...so just need to make sure we won't accidentally pick up runtimes that isn't in `Deleting`